src/domains/idpay-app/env/prod/terraform.tfvars

```
@@ -60,6 +60,11 @@ pdv_timeout_sec   = 5
 pm_service_base_url = "https://api-io.cstar.pagopa.it"
 pm_backend_url      = "https://api.platform.pagopa.it"
 
+#
+# MIL
+#
+openid_config_url_mil = "https://mil-u-apim.azure-api.net/mil-auth/.well-known/openid-configuration"
```

Let's ask what will be PROD value
```suggestion
openid_config_url_mil = "https://mil-p-apim.azure-api.net/mil-auth/.well-known/openid-configuration" #TODO to verify
```